components/applications-service/pkg/storage/postgres/service_internal_test.go

```diff
@@ -4,9 +4,39 @@ import (
 	"regexp"
 	"testing"
 
+	"github.com/chef/automate/components/applications-service/pkg/storage"
 	"github.com/stretchr/testify/assert"
 )
 
+func TestConvertComposedServiceToStorage(t *testing.T) {
+	expected := &storage.Service{}
+	assert.Equal(t, expected, convertComposedServiceToStorage(&composedService{}))
+
+	expected = &storage.Service{ID: 1, SupMemberID: "foo"}
+	assert.Equal(t, expected, convertComposedServiceToStorage(&composedService{ID: 1, SupMemberID: "foo"}))
+
+	expected = &storage.Service{Application: "abcd", PreviousHealth: "MORE-OR-LESS-OK"}
+	assert.Equal(t, expected, convertComposedServiceToStorage(&composedService{
+		Application: "abcd", PreviousHealth: "MORE-OR-LESS-OK", Status: "",
+	}))
+}
+
+func TestConvertComposedServicesToStorage(t *testing.T) {
+	expected := []*storage.Service{
+		&storage.Service{},
+		&storage.Service{ID: 1, SupMemberID: "foo"},
+		&storage.Service{Application: "abcd", PreviousHealth: "MORE-OR-LESS-OK"},
+	}
+	subject := convertComposedServicesToStorage([]*composedService{
+		&composedService{},
+		&composedService{ID: 1, SupMemberID: "foo"},
+		&composedService{Application: "abcd", PreviousHealth: "MORE-OR-LESS-OK", Status: ""},
+	})
+	for _, e := range expected {
```

Comment:
❓ How is this different from `assert.ElementsMatch(t, expected, subject)`? I might need more coffee to see it.

Reply:
I have no idea but it didn't work 😂 - I might need more ☕️ as well